intel_extension_for_transformers/llm/runtime/graph/models/chatglm/chatglm2.cpp

```diff
@@ -11,6 +11,8 @@
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
+#include "chatglm2.h"
```

Comment:
confusing, we don't have this before? @Zhenzhong1

Reply:
Yes. chatglm2.h is included in the chatglm2_utils.cpp. The function declarations are used by chatglm2_utils.cpp but not chatglm2.cpp directly.